slf4j/src/main/scala/zio/logging/slf4j/Slf4jLogger.scala

```diff
@@ -1,13 +1,15 @@
 package zio.logging.slf4j
 
+import izumi.reflect.Tag
```

Comment:
```suggestion
import zio.Tag
```

It's just an alias though :).